llpc/translator/lib/SPIRV/SPIRVReader.h

```diff
@@ -315,6 +316,23 @@ class SPIRVToLLVM {
                                              std::function<Value *(Value *)> createImageOp);
 
   Function *createLibraryEntryFunc();
+
+  const Vkgc::PipelineOptions *getPipelineOptions() const;
+
+  // This flag must explicitly be updated while translating if we want to continue inserting at the last insert point
+  // instead of omitting it.
+  bool continueAtLastInsertPoint = false;
+
+  template <class MemOpType> SPIRVValue *getValueForScratchBoundsCheckWrapper(MemOpType *toBeWrapped) const;
+  struct ScratchBoundsCheckInfo {
+    bool shouldInsertBoundsCheck = false;
```

Comment:
Since there are no special initial values for each field, we don't need explicitly assign them to nullptr or false. If we need zero the structure, we can make this: `struct ScratchBoundsCheckInfo checkInfo = {}`